flake.nix

```diff
@@ -0,0 +1,76 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachSystem [
+      "x86_64-linux"
+    ] (system:
+      let
+        pkgs = import nixpkgs {
+          inherit system;
+        };
+
+        libPath = pkgs.lib.makeLibraryPath [
+          pkgs.wayland
+          pkgs.xorg.libX11
+          pkgs.xorg.libXcursor
+          pkgs.xorg.libXext
+          pkgs.xorg.libXi
+          pkgs.xorg.libXrandr
+        ];
+
+        buildDependencies = [
+              pkgs.pkg-config
+
+              # build system
```

Comment:
would it make sense to add a compiler?

Reply:
I prefer gcc, but mingw and emcc are needed for the other platforms yoyoengine supports

Reply:
A compiler (`gcc`) is automatically added as part of the `stdenv`.